clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -98,6 +98,19 @@
                                                          + "elapses the client will resend the request if necessary or fail the request if "
                                                          + "retries are exhausted.";
 
+    public static final String DEFAULT_LIST_KEY_SERDE_INNER_CLASS = "default.list.key.serde.inner";
```

Comment:
Seems this variable is still misssing the corresponding DOC string? (Same for `DEFAULT_LIST_VALUE_SERDE_INNER_CLASS` below)